userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -1580,6 +1633,125 @@ int32_t scap_bpf_get_stats(struct scap_engine_handle engine, OUT scap_stats* sta
 	return SCAP_SUCCESS;
 }
 
+struct scap_stats_v2* scap_bpf_get_stats_v2(struct scap_engine_handle engine, uint32_t flags, OUT uint32_t* nstats, OUT int32_t* rc)
+{
+	struct bpf_engine *handle = engine.m_handle;
+	int ret;
+	int fd;
+	int i = 0; // offset in stats
```

Comment:
Let's rename the variable rather than add a comment? :) `i` outside a single loop is confusing.